docs/SemanticARC.md

```diff
@@ -376,12 +434,19 @@ generalized dominators that can be used. If computation of generalized
 dominators is too expensive for normal use, they could be used on specific
 verification bots and used when triaging bugs.
 
+[EE]
+When looking at semantic ARC in terms of local RCs (see my comment above), the verifier is very simple. It's a dataflow algorithm, which ensures that the local RCs are consistent at block entries for all RC roots. It's similar to what we do to check proper nesting of alloc_stacks.
```

Comment:
That is essentially what I am proposing, but not just block entries. You also have to check for store_strongs and the like which are not block entries.